analyzers/tests/SonarAnalyzer.UnitTest/LiveVariableAnalysis/RoslynLiveVariableAnalysisTest.cs

```diff
@@ -777,22 +779,6 @@ public void ProcessVariableInForeach_Declared_LiveIn_LiveOut()
             context.ValidateExit();
         }
 
-        [TestMethod]
-        public void ProcessVariableInForeach_Reused_LiveIn_LiveOut()
-        {
-            var code = @"
-int i = 42;
-foreach(i in new int[] {1, 2, 3})
-{
-    Method(i, intParameter);
-}";
-            var context = CreateContextCS(code);
-            context.Validate(context.Cfg.Blocks[1], null, new LiveIn("intParameter"), new LiveOut("intParameter"));
-            context.Validate(context.Cfg.Blocks[2], null, new LiveIn("intParameter"), new LiveOut("intParameter"));
-            context.Validate("Method(i, intParameter);", new LiveIn("intParameter"), new LiveOut("intParameter"));
-            context.ValidateExit();
-        }
```

Comment:
Foreach variable cannot be reused => this UT doesn't make sense. There's another one to test normal `foreach`